Main/tests/ZombieWalkerTest.kt

```diff
@@ -0,0 +1,12 @@
+import org.junit.jupiter.api.Test
+import kotlin.test.assertEquals
+
+internal class ZombieWalkerTest {
+    @Test
+    fun `should return string with ZombieWalker`(){
+        val controlSpeed = 10
+        val actual = ZombieWalker()
```

Comment:
Actual must be final. But u'r use in the assert actual.walk
It's bad practice
Move calling method to actual initialisation